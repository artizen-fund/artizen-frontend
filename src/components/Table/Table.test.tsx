import { render } from '@testing-library/react'
import { Table } from './'

it('renders Table unchanged', () => {
  const { container } = render(<Table title="derp" />)
  expect(container).toMatchSnapshot()
})
