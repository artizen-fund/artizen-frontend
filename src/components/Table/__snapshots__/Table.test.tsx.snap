// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Table unchanged 1`] = `
<div>
  <div
    class="sc-8fd18655-0 bELACC"
  >
    <div
      class="sc-c6d4a71-0 iFQvvL"
    >
      <div
        class="sc-c6d4a71-1 hmYVNc"
      >
        derp
      </div>
    </div>
  </div>
</div>
`;
